meson.build

```diff
@@ -4,12 +4,12 @@
 project(
     'OpenDCDiag',
     ['c', 'cpp'],
-    default_options : [
+    default_options: [
```

Comment:
Clean ups are correct but noise here. Can you submit a separate patch for them?

Reply:
Yes, you're right, I need to refrain from auto-linting these files.